lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnJobSubmissionHandler.java

```diff
@@ -90,19 +93,23 @@ public void close() {
   @Override
   public void onNext(final JobSubmissionEvent jobSubmissionEvent) {
 
-    LOG.log(Level.FINEST, "Submitting job with ID [{0}]", jobSubmissionEvent.getIdentifier());
+    LOG.log(Level.FINER, "Submitting {0} job with ID {1}",
+        new Object[] {this.isDriverUnmanaged ? "UNMANAGED" : "regular", jobSubmissionEvent.getIdentifier()});
```

Comment:
Maybe harmonize this to `unmanaged` and `managed`? `regular` tends to loose meaning over time.